.gitignore

```diff
@@ -2,4 +2,4 @@
 /tests/cucumber/node_modules/
 .vagrant/*
 autogenerated_by_travis.sh
```

Comment:
Removed the travis reference in this commit: b54039bdbefff5cc6e6fe0542d3289aaad6970a9